parent/pom.xml

```diff
@@ -1214,6 +1215,13 @@
         <artifactId>jakarta.activation-api</artifactId>
         <version>${version.jakarta-activation}</version>
       </dependency>
+
+      <!-- needed for tests in elasticsearch-exporter -->
+      <dependency>
+        <groupId>jakarta.json</groupId>
+        <artifactId>jakarta.json-api</artifactId>
+        <version>${version.jakarta.json-api}</version>
+      </dependency>
```

Comment:
This method [`lifecycle.policy().phases().delete().actions().toJosn()`](https://github.com/camunda/zeebe/blob/e3b0d88f72cfa6241b614d67e42fb11de56644e9/exporters/elasticsearch-exporter/src/test/java/io/camunda/zeebe/exporter/ElasticsearchClientIT.java#L185C1-L186C1) depends on `jakarta.json.JsonValue` from `jakarta.json-api`. It also uses jackson under the hood.

I also understand, adding a dependency just for 1 line of test code is silly. 
The workaround is to use Jackson ObjectMapper and parse/traverse the output of `lifecycle.policy().phases().delete()`. 

I tried it. it is a bit hacky, ugly and require some manipulation since the output of the `Phase` is not standard JSON 
```
Phase: {"actions":{"delete":{"delete_searchable_snapshot":true}},"min_age":"30d"}
```